### PR TITLE
Add an "Adding Additional Repositories" section to the getting-started docs.

### PR DESCRIPTION
**Pull request by Dvlv**
Hello, I have added a section about how to add additional repositories, since it has been brought up in the Matrix chat a couple of times.

### CODE REVIEW

File: modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -128,3 +128,12 @@ You may also use `override remove` to effectively "hide" packages; they will sti
...
 Removing and replacing packages using package layering is not generally recommended.
 For more information, see the https://coreos.github.io/rpm-ostree/administrator-handbook/[rpm-ostree documentation].
+
+=== Adding Additional Repositories
+
+Some software may require adding a third-party repository. This can be achieved by downloading the `.repo` file into `/etc/yum.repos.d/`. 
+For example, to install Tailscale, you can do the following:
+
+ $ cd /etc/yum.repos.d/
+ $ sudo wget https://pkgs.tailscale.com/stable/fedora/tailscale.repo
```
Comment (travier) [inline modules/ROOT/pages/getting-started.adoc:138]: Can you use a `curl | sudo tee` command here to avoid doing networking as root?

**Review by travier — COMMENTED** [review]
Mostly LGTM. One small nit. Thanks!

**Comment by Dvlv** [comment]
Thanks @travier, I've changed that.

**Review by travier — APPROVED** [review]

**Comment by travier** [comment]
Thanks!

**Pull request merged**